dev-manuals/test-plans/minigames/bugfinder-frontend/plans/u-bugfinder-6-next-code.md

```diff
@@ -0,0 +1,54 @@
+# Bugfinder next code (u.bugfinder-6)
+
+
+Version: V1.0, 05.08.2022
+Author: Timo Schnaible
+Tester: -
+
+## Description
+
+When the player submited his solution he can get to the next code snipped by pressing the `Next Code` button which replaced the `Submit` button. The button is only shown if there is another code snipped. Additionally the virtual chat partner asks for help with another bug in the chat.
+
+## Precondition
+
+The tester submited and another code snipped exists.
+
+## Postcondition
+
+The next code snipped is shown.
+
+## Typical procedure
+
+1. Press start to start the game
+2. The first code snipped is shown
+3. Select the bugs
+4. Press the `Submit` button
+5. The `Submit` button gets replaced by the `Next Code` button
```

Comment:
Precondition. Maybe apart from ``The `Submit` button is now a `Next Code` button`` here.